lists2safebrowsing.py

```diff
@@ -614,12 +615,14 @@ def get_versioned_lists(config, chunknum, version):
             ver=version, output=config.get(section, 'output'))
         )
         version_configurations(config, section, version)
+        ver = p_version.parse(version)
         if (section in PRE_DNT_SECTIONS or section in DNT_SECTIONS):
+            if ver.release[0] < VERSION_EMAIL_CATEGORY_INTRODUCED:
+                continue
```

Comment:
This looks like it will skip all sections in `PRE_DNT_SECTIONS` and `DNT_SECTIONS` for versions older than 97, and not only the new email-related sections. This might be what's intended – I don't fully understand the context here. :)

Reply:
Ack, you are right. This was only supposed to apply for `Email` and `EmailStrict` category. Let me update.

Reply:
Fixed: e651dbce1bf70e257bddba7bcfb9aeb1ac470767